test-harness/build.gradle

```diff
@@ -26,3 +26,23 @@ task sourcesJar(type: Jar) {
 artifacts {
   archives jar, javadocJar, sourcesJar
 }
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      pom.project {
+        name 'Test Harness for SpotBugs Plugin'
+        description 'A test harness library for SpotBugs plugin developers'
+
+        licenses {
+          // This project contains copied source files from SonarSource and find-sec-bugs,
+          // which is licensed under LGPL 3.0, so we should use v3.0 instead of 2.1.
```

Comment:
:+1: I guess that counts as written authorization :P. I guess this is the only change then.